src/components/SelectionList/CardListItem.tsx

```diff
@@ -77,6 +79,13 @@ function CardListItem<TItem extends ListItem>({
                                 item.alternateText ? styles.mb1 : null,
                             ]}
                         />
+                        {!!item.lastFourPAN && (
+                            <TextWithTooltip
+                                shouldShowTooltip={showTooltip}
+                                text={translate('subscription.cardSection.cardEnding', {cardNumber: item.lastFourPAN})}
```

Comment:
```suggestion
                                text={`${translate('paymentMethodList.accountLastFour')} ${item.lastFourPAN}`}
```

@daledah As I noted earlier in https://github.com/Expensify/App/pull/47773#discussion_r1725970545, another PR https://github.com/Expensify/App/pull/47757 just merged the translation that we need here as well, please sync with main to pull the new translations and then use this suggestion to adjust the translation. Once this is done we're ready for merge! 🟢